WordPress/src/main/java/org/wordpress/android/ui/plugins/PluginDetailActivity.java

```diff
@@ -493,10 +494,8 @@ public void onPluginInfoChanged(PluginStore.OnPluginInfoChanged event) {
                     + event.error.type);
             return;
         }
-        if (event.pluginInfo != null
-                && mPlugin.getSlug() != null
-                && mPlugin.getSlug().equals(event.pluginInfo.getSlug())) {
-            mPluginInfo = event.pluginInfo;
+        if (!TextUtils.isEmpty(mPlugin.getSlug()) && mPlugin.getSlug().equals(event.pluginSlug)) {
+            mWPOrgPlugin = mPluginStore.getWPOrgPluginBySlug(event.pluginSlug);
```

Comment:
I'm not sure about this change. Why get the plugin from the store when it's in the event?

Reply:
It's not in the event anymore and that's by design. I think our approach is, when we can pull the same information from the store, we don't include it in the event.

I didn't do it the correct way before and took this opportunity to fix it. I don't think it mattered much before, but since from now on, we'll have the same information coming from multiple sources, it's important to have one source of truth for WPAndroid which will be the `getWPOrgPluginBySlug`. Hope that makes sense!